src/f256.rs

```diff
@@ -0,0 +1,35 @@
+
+pub const FIELD_NAME: &'static str = "f256";
+
+pub type Elt = u8;
+pub type Wide = u16;
+
+pub const FIELD_BITS: usize = 8;
+pub const FIELD_SIZE: usize = 1_usize << FIELD_BITS;
+
+include!("f_log_mul.rs");
+
+#[cfg(table_bootstrap_complete)]
+include!("afft.rs");
```

Comment:
No!

Reply:
I do think `pub(crate) struct AdditiveFFT<F: BinaryField>` could exist without this since `AdditiveFFT::afft`/`inverse_afft` require relatively little too, at least initially though maybe later maybe they require macros.

`AdditiveFFT::initialize` should definitely be a macro or use `include!` though because we'll yank the field representation out from under it I think.  If you read my comments in https://github.com/drahnr/rs-ec-perf/pull/11/files#diff-f3f3b9070bc5ad37ca82632b2043fe553dcade424b615a4138478e105dcdde61R253-R275 then you'll see we do not currently understand enough to change it though.